src/language/tekster.ts

```diff
@@ -713,10 +713,10 @@ export default {
     'søkerFraBestemtMåned.hjelpetekst-åpne':
       'Om å søke fra et bestemt tidspunkt',
     'søkerFraBestemtMåned.hjelpetekst-innhold.overgangsstønad':
-      'Du kan få overgangsstønad fra og med måneden etter at du har rett til stønaden.' +
-      'Du kan ha rett til stønad i inntil 3 måneder før du søker. Det vil si fra og med april 2020.' +
-      'Hvis du er gravid, kan du ha rett til overgangsstønad fra måneden før fødsel.' +
-      'Hvis du har fått barn i løpet av de siste 3 månedene, kan du få stønad i inntil 5 måneder før du søker. Det vil si fra og med februar 2020.' +
+      'Du kan få overgangsstønad fra og med måneden etter at du har rett til stønaden. ' +
+      'Du kan ha rett til stønad i inntil 3 måneder før du søker. Det vil si fra og med april 2020. <br><br>' +
+      'Hvis du er gravid, kan du ha rett til overgangsstønad fra måneden før fødsel. ' +
+      'Hvis du har fått barn i løpet av de siste 3 månedene, kan du få stønad i inntil 5 måneder før du søker. Det vil si fra og med februar 2020. <br><br>' +
```

Comment:
Er det riktig at april/februar 2020 skal være hardkodet her?

Reply:
Jeg sjekket koden, og den har egentlig alltid vært hardkodet.. Selv om det ser ut som at `hentBeskjedMedFireParametre` skal tweake på teksten, så er output helt det samme som den får inn siden selve tekststrengen ikke hadde [0], [1], [2] og [3] som definerer hva som skal tweakes. 

Det var også litt tricky å få formattert teksten med `<br/>` etc.
Er litt vanskelig med de tekstene som skal ha innfletta ord og liknende, samtidig formatteres på det spesiell måte og være uu 🙈  

Fordi for å kunne formattere inni tekstene bruker vi html tags, og for at hmtl tagsa skl kunne leses av intl, bruker vi `<FormattedHTMLMessage id={id} />`. Den må ta inn en tekstid. 

Hvis jeg skulle gjort no slik som den gamle koden, bare at vi erstatter hentTekst med `intl.formatHTMLMessage`:
```
  const hjelpetekst = hentBeskjedMedFireParametre(
    intl.formatHTMLMessage({ id: hjelpetekstInnholdTekstid}),
    '3',
    formatNårSøkerDuStønadFraMåned(dagensDato, 3),
    '5',
    formatNårSøkerDuStønadFraMåned(dagensDato, 5)
  );
```

Så går det ikke, siden formatHTMLMessage returnerer en React.Node. 
Så jeg får ikke oversatt den før jeg tweaker på teksten pga dette, og jeg får heller ikke formattert teksten med formatHTMLMessage etter å ha tweaket på teksten fordi jeg trenger en tekstid og ikke klar tekst. 

Det å å hardkode var komprimisset jeg endte opp med etter å ha oppdaget at `hentBeskjedMedFireparametere` ikke ble brukt uansett. Inntil vi finner noe bedre.. 

Tar gjerne imot innpsill og forslag på dette

Reply:
Hmm, det er litt hacky, men det vi kunne gjort er å delt teksten inn i 3 tekster (en tekst per avsnitt). Da vil vi ikke trenge å formattere teksten, og da kan man flette inn datoene.

Reply:
ny commit med dynamisk mnd og dato eller å ha delt teksten i 3 `4921de1`